app/controllers/errors_controller.rb

```diff
@@ -8,7 +8,7 @@ def not_found
   end
 
   def internal_server_error
-    render_error("internal_server_error", :internal_server_error)
+    render_error(nil, :internal_server_error)
```

Comment:
what do you think of rendering the static file here (with send_static)? The `send_static_error_page` handles the case were 500.html does not exist, I think we can safely assume it exists